cost_sensitive/main.py

```diff
@@ -117,16 +108,12 @@ def prepare_clfs(weights):
 
```

Comment:
Jeśli wszystkie datasety są teraz binarne i nie korzystamy z roc to można wywalić te `predict_proba`